packages/@orchidui-vue/src/Form/Select/OcOption.vue

```diff
@@ -21,11 +36,24 @@ defineProps({
       :is-partial="isPartial"
       class="!w-fit pointer-events-none"
     />
-    <div class="flex flex-col gap-3" :class="{ 'flex-1': isCheckboxes }">
-      <span>{{ label }}</span>
-      <span v-if="subLabel" class="text-sm text-oc-text-300">{{
-        subLabel
-      }}</span>
+    <div class="flex items-center gap-x-3">
+      <div
+        v-if="image"
+        class="aspect-square rounded w-10 border border-oc-accent-1-100"
+      >
+        <img
+          :src="image"
+          :alt="label"
+          class="object-contain h-full w-full"
+          loading="lazy"
+        />
+      </div>
+      <div class="flex flex-col gap-3" :class="{ 'flex-1': isCheckboxes }">
```

Comment:
Please use slot. it should be safe & keep the component is cleanliness & more maintainable